### ISSUE TITLE
Typescript Playground misinterprets `<Type>value` as React tag?

### ISSUE THREAD

**Issue by grapevinegizmos** [microsoft/TypeScript-Website#1370]
**TypeScript Version:**  4.1.2

**Search Terms:**
Playground interface 

**Code**
```
interface Person {
    name: string;
    age: number;
}

const makePerson= function(name : string, age : number) : Person {
    
    let obj = <Person>{}
    obj.name = name;
    obj.age = age;
    return obj  
} 
```

 

**Expected behavior:**

Should compile with no errors.  (Or at least, this syntax works in VSCode with Typescript v 4.0.3.)

**Actual behavior:**

Produces syntax error as per screenshot::  
![image](https://user-images.githubusercontent.com/8684037/100279209-f3bf7c80-2f1a-11eb-97dd-db91bb992cb0.png)

The transpiled javascript doesn't look right, seems to call `React.createElement` for some reason.

**Playground Link:** 
https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgArQM4HsTIN4CwAUMqciHALYQBcyGYUoA5gNzFnJzO3kCulAEbR2RAL7FiCHA2SU4Aawjoo2EAF5kMPiARhgOABQVqyOgyYhmAGi48z-IdACUDlWvwcyX0gBsIYMhYggBWyJoAPO44AHx4EiRkwSEAdCYomuminMkp3Bl2ENlkUAF8ULjJpMQJQA
**Related Issues:**

**Comment by orta** [comment]
You can disable using .tsx by setting react to "none" in the compiler options dropdown: https://www.typescriptlang.org/play?jsx=0#code/JYOwLgpgTgZghgYwgAgArQM4HsTIN4CwAUMqciHALYQBcyGYUoA5gNzFnJzO3kCulAEbR2RAL7FiCHA2SU4Aawjoo2EAF5kMPiARhgOABQVqyOgyYhmAGi48z-IdACUDlWvwcyX0gBsIYMhYggBWyJoAPO44AHx4EiRkwSEAdCYomuminMkp3Bl2ENlkUAF8ULjJpMQJQA

👍🏻

**Comment by grapevinegizmos** [comment]
You know, I suspected this might solve the problem, (using react: none), when I encountered this, but changing the setting had no obvious effect.

I just tried it again, and see that if I refresh the page the error disappears, but I"m not sure that users would understand that in order to force the playground to adopt the new setting a refresh is required.  (Actually as a newbie I was afraid of refreshing for fear I'd lose all my code.). Suggest you might want to address that.
